### ISSUE TITLE
IBM Cloud Admin documentation has been removed

### ISSUE THREAD

**Issue by rigoletj** [ibm-cloud-docs/cli#105]
The documentation for the dedicated and local admin console that used to be at https://cloud.ibm.com/docs/admin/index.html is no longer available.
As this is still used by dedicated and local consoles, could you set it back please?
Thank you.

**Comment by jpkessle** [comment]
@rigoletj Is the documentation you are looking for in this topic:

IBM Cloud admin CLI
https://cloud.ibm.com/docs/cli?topic=cloud-cli-ibmcloud-admincli

**Comment by rigoletj** [comment]
@jpkessle no, I'm looking for the main Admin Console documentation at https://cloud.ibm.com/docs/admin/index.html which used to exist and seems to have vanished.

**Comment by jpkessle** [comment]
@rigoletj -
I am not sure what happened to this content as I don't own it, however, I will do some further investigation and provide an update on this before the end of the day.
Thank you!
Josh

**Comment by jpkessle** [comment]
@rigoletj It appears the `/docs/admin` content was removed so long ago that we are unable to pinpoint exactly when it was brought offline. I suspect the link you have was a very old bookmark? As we currently do not use `*.html` style URLs any longer since we have moved to a new topicID system last year.

The Admin Console related documentation still available deals with CLI and API:

IBM Cloud admin CLI
https://cloud.ibm.com/docs/cli?topic=cloud-cli-ibmcloud-admincli

Managing Bluemix Local and IBM Cloud Dedicated
https://cloud.ibm.com/docs/hybrid?topic=hybrid-mng#auth_adminapi

I hope this helps. 
Thank you!
Josh
